opennms-config/src/main/resources/META-INF/opennms/applicationContext-commonConfigs.xml

```diff
@@ -166,4 +166,15 @@
 
   <onmsgi:service interface="org.opennms.netmgt.config.api.ResourceTypesDao" ref="resourceTypesDao" />
 
+	<bean id="contextServiceMonitorLocator" class="org.opennms.netmgt.config.ContextServiceMonitorLocator">
```

Comment:
Delete this bean. We'll only be creating ContextServiceMonitorLocators inside PollerConfigManager.